zenoh/src/api/queryable.rs

```diff
@@ -534,43 +538,13 @@ impl fmt::Debug for QueryableState {
     }
 }
 
-/// An entity able to reply to queries through a callback.
```

Comment:
Even if the code has changed, I believe the doc is still relevant when pointing to the right type and with an appropriate description.

Reply:
What would you write here?

Reply:
Nevermind, after a second read I believe the content is covered in the new comments.